fpdf/fpdf.py

```diff
@@ -1054,6 +1054,41 @@ def circle(self, x, y, r, style=None):
         """
         self.ellipse(x, y, r, r, style)
 
+    @check_page
+    def regular_polygon(self, x, y, numSides, polyWidth, rotateDegrees=0, style=None):
+        """
+        Outputs a regular polygon with n sides
+        It can be rotated
+        Style can also be applied (fill, border...)
+
+        Args:
+            x (int): Abscissa of upper-left bounging box.
+            y (int): Ordinate of upper-left bounging box.
```

Comment:
@bettman-latin, probably just an oversight: The correct spelling would be "bounding box"... :wink: